runtime/doc/lua.txt

```diff
@@ -94,8 +96,8 @@ Note:
 
 - Skipping paths from 'runtimepath' which contain semicolons applies both to
   `package.path` and `package.cpath`.  Given that there are some badly written
-  plugins using shell which will not work with paths containing semicolons it
-  is better to not have them in 'runtimepath' at all.
+  plugins using shell, which will not work with paths containing semicolons,
```

Comment:
do not add commas everywhere. commas are intentionally used sparingly. This is English, not German.